src/Statement/StatementList.php

```diff
@@ -105,6 +106,34 @@ public function addNewStatement( Snak $mainSnak, $qualifiers = null, $references
 		$this->addStatement( $statement );
 	}
 
+	public function removeStatement( Statement $statement ) {
```

Comment:
Actually I think it is best to create a dedicated PR for adding that method and one for the `clear` one. So a -2 for one method does not block the other